src/main/java/com/example/SimpleConnection.java

```diff
@@ -52,20 +52,6 @@ private void connect(String hostname, int port, String userName, String password
         ExperimenterData user = gateway.connect(cred);
         ctx = new SecurityContext(user.getGroupId());
     }
-
-    /** 
-     * Creates a connection, the gateway will take care of the services
-     * life-cycle.
-     *
-     * @param args The arguments used to connect.
-     */
-    private void connect(String[] args)
-        throws Exception
-    {
-        LoginCredentials cred = new LoginCredentials(args);
```

Comment:
Is it also worth adding a caveat to the Javadoc of https://github.com/openmicroscopy/openmicroscopy/blob/v5.4.0-m2/components/blitz/src/omero/gateway/LoginCredentials.java#L76?